### ISSUE TITLE
Error 500 when i try to install xibo 1.8.8

### ISSUE THREAD

**Issue by luxas0072** [xibosignage/xibo#1476]
Hello everyone i have a ubuntu 16.04.4 with xibo 1.7.1 and no docker, i did a manual installation
My xibo 1.7.1 was working fine expect that the dashboard wasn't working correctly (https://gyazo.com/334a9752ef7d7c62814cedcf119e66e2) and the web region didn't show anything.
Here the settings of my web region:https://gyazo.com/6eb894ac371121d93ee6f88cfe22969d https://gyazo.com/cdbf9640e35b3533a9fa1023f6ddbc3e
And the result with everything but the website region: https://gyazo.com/5e833a617bb58f3b63fc2263dc06885d

So i though upgrading the version of xibo would fix the problem so i followed the xibo manual guide: https://xibo.org.uk/manual/en/upgrade_custom.html
And when i try to go to the xibo page localhost/xibo/web i get redirected in localhost/xibo/web/install but there is a error 500 that says that the website isn't working properly: https://gyazo.com/75920353d1c9e1753f67fc80c85e8c7b

I saw also that compare to version 1.7.1 where i had to go to localhost/xibo now i have to go to localhost/xibo/web that might be a compatibily issues with the settings i don't really know

TL:DR
So i have two options:

- Fix the 2 bug with version 1.7.1 dashboard+website region
- Fix the localhost/xibo/web/install error 500

Fixing the error 500 might be the easiest option to fix

Anyway thanks for your help.
Have a nice day!

**Comment by dasgarner** [comment]
Please ask your question on our [community forum](https://community.xibo.org.uk) - this repository is for verified bugs in Xibo only (as stated in the contributing notice that you replaced with your message).

Thanks.
